CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ was committed to the `master` branch. This is not necessarily the date that
 the code was deployed.
 
 ## [Unreleased]
+### Added
+- Send urgent messages if it has been more than 2 minutes since last button press, regardless of if it is a multiple of 5 presses (CU-j6y98q).
```

Comment:
Yes - do you think I should say since last session update maybe though since that is more accurate?

(Summary of why for other people: the reminder and fallback messages both also update the session so if someone presses for a 3rd or 4th time right after those updates those presses will be lost as far as having their own alerts goes, but there will still be 2 alerts 2 minutes apart in that 4 minute window and any presses that happen >= 2 minutes after the fallback will send an alert if it's been 2 minutes since the last one regardless of if they're a multiple of 5)